tasks.py

```diff
@@ -26,13 +17,23 @@ def deploy_nixos(hosts: List[DeployHost]) -> None:
     g = DeployGroup(hosts)
 
     def deploy(h: DeployHost) -> None:
-        target = f"{h.user or 'root'}@{h.host}"
         h.run_local(
-            f"rsync {' --exclude '.join([''] + RSYNC_EXCLUDES)} -vaF --delete -e ssh . {target}:/etc/nixos"
+            [
+                "nixos-rebuild",
+                "switch",
+                "--option",
+                "accept-flake-config",
+                "true",
+                "--flake",
+                f".#{h.host}",
+                "--fast",
+                "--build-host",
+                f"{h.host}",
```

Comment:
If you set a build host it will cause all derivations to be uploaded. This is useful for cross-arch deployments but otherwise it's significant slower than a normal nixos-rebuild.

```suggestion
                f"",
```

Reply:
Wouldn't we need this for x86_64-linux -> aarch64-linux or darwin -> linux deployments?